src/main/java/rs/teslaris/core/controller/PersonController.java

```diff
@@ -38,6 +38,12 @@ public class PersonController {
 
     private final PersonIndexService personIndexService;
 
+    
+    @GetMapping
+    public Page<PersonIndex> findAll(Pageable pageable) {
+        return personIndexService.findAll(pageable);
+    }
+
```

Comment:
It looks to me it will be better encapsulated if we create one PersonService which is ingested in PersonController which is communicating with  ES and/or JPA repositories. For instance, for DocumentServiceImpl there is communication with FileStorage and JPA repository, I am not sure why there shouldn't be also communication with ES. In that way, Services are DAOs. So, my suggestion is not to separate PersonService and PersonIndexService